src/utils/get.js

```diff
@@ -0,0 +1,15 @@
+async function getData() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/rO378WlzTyLm1i3wuLMC/scores/');
+  const json = await response.json();
+  return json;
+}
+
+export default function renderData() {
+  const ul = document.querySelector('.score-list');
+  getData().then((json) => {
+    const array = json.result;
+    for (let i = 0; i < array.length; i += 1) {
+      ul.innerHTML += `<li class="score">${array[i].user} ${array[i].score}</li>`;
+    }
+  });
+}
```

Comment:
- [ ] Kindly make use of `arrow functions` instead of the `function` keyword to follow ES6 best practices.